lang/cs/Org.Apache.REEF.Network/Group/Operators/IGroupCommOperator.cs

```diff
@@ -39,5 +39,7 @@ public interface IGroupCommOperator<T>
         /// The operator version number.
         /// </summary>
         int Version { get; }
+
+        void Initialize();
```

Comment:
Is it to ensure all tasks are registered before send/receive messages. The method name is little big confusion though. 
It cannot be in the constructor. Because the operator constructor is injected within task constructor (nested), here we are waiting for all tasks to be registered. In the test  case, it can be dead lock. In real scenarios, since tasks are running on different VMs, they are essentially running in parallel, it might be OK.